### DISCUSSION TITLE
Is there any document for use neon database with s3 api oss ( for example minio ) ?

### DISCUSSION THREAD

**Discussion by usrtax** [neondatabase/neon#2370] [Q&A]
Is there any document for use neon database with s3 api oss ( for example minio ) ?

maybe give a docker-compose.yml example is easy for beginner to use

**Comment by raoufchebri** [comment]
Hi @usrtax, we haven't documented it yet because it is not ready to be used publicly. Please stay tuned, and we'll let you know as soon as the doc is available.

**Comment by mingfang** [comment]
I was able to get Neon working with LocalStack.
The trick is to set these env variables for the Safekeeper and Pageserver services.
```
  S3_ENDPOINT="http://localstack:4566"
  AWS_ACCESS_KEY_ID="test"
  AWS_SECRET_ACCESS_KEY="test"
```
Also make sure to create a bucket named `neon` before starting.
Minio should work the same way.
